src/test/scala/cornichonTests/AuthTest.scala

```diff
@@ -1,25 +1,18 @@
 package cornichonTests
+
 import com.github.agourlay.cornichon.CornichonFeature
-import eu.timepit.refined.api.Refined
-import eu.timepit.refined.pureconfig._
-import eu.timepit.refined.string.Url
-import pureconfig.generic.auto._
-import pureconfig._
```

Comment:
Those changes are not related to formatting right?
And are you sure the project compiles without those imports?

Reply:
as discussed face-to-face, those imports are not necessary